src/js/plugin/external/PluginHelper.js

```diff
@@ -0,0 +1,27 @@
+import PluginDispatcher from "./PluginDispatcher";
+
+var PluginHelper = {
+  callAction: function (action, data) {
+    PluginDispatcher.dispatch({
+      actionType: action,
+      data: data
+    });
+  },
+
+  injectComponent: function (component, placeId) {
+    PluginDispatcher.dispatch({
+      eventType: "INJECT_COMPONENT",
```

Comment:
I am sure I've used the constant, strange. Maybe I've forgot to push on my other machine.